modules/photon-cdn.php

```diff
@@ -151,7 +151,11 @@ public static function cdnize_plugin_assets( $plugin_slug, $current_version ) {
 	 * @return array
 	 */
 	public static function get_plugin_assets( $plugin, $version ) {
-		if ( 'jetpack' === $plugin && JETPACK__VERSION === $version ) {
+		if (
+			'jetpack' === $plugin
+			&& JETPACK__VERSION === $version
+			&& self::is_public_version( $version )
```

Comment:
If it's not a public version of Jetpack, we'll end up failing this condition and proceeding to the (cached) plugins repo check below.

Since we know no non-public version of Jetpack will have assets in the plugins repo, we can bail early.

How about removing this condition from the `if` statement, and adding an:
```
if ( ! public ) {
  return array();
}
```

or whatever inside the current `if` statement.